Source/UserManagement/Domain/StaffUser/Registering/RegisterNewStaffDataVerifier.cs

```diff
@@ -4,18 +4,17 @@
 
 namespace Domain.StaffUser.Registering {
 
-    public class RegisterNewStaffDataVerifier : NewStaffRegistration<Roles.DataVerifier>, IRequireLocation,
-        IRequireBirthYear, IRequirePreferredLanguage, IRequireSex, IRequireNationalSociety, IRequirePhoneNumbers 
+    public class RegisterNewStaffDataVerifier : NewStaffRegistration<Roles.DataVerifier>
     {
         public RegisterNewStaffDataVerifier () 
         {
             Role = new Roles.DataVerifier();    
         }
```

Comment:
@smithmx This Michael, this must have been some leftover stuff?

Reply:
Looks like it! :-)